pkg/controller/keepalivedgroup/keepalivedgroup_controller.go

```diff
@@ -253,7 +270,6 @@ func assignRouterIDs(instance *redhatcopv1alpha1.KeepalivedGroup, services []cor
 	for _, value := range toBeRemovedSet.List() {
 		delete(instance.Status.RouterIDs, value)
 	}
-	assignedIDs := []int{}
 	for _, value := range instance.Status.RouterIDs {
 		assignedIDs = append(assignedIDs, value)
 	}
```

Comment:
After this loop, sort assignedIDs and remove duplicates

Reply:
sorting is useless, no duplicates are expected to exist.

Reply:
The sort is to make the next assigned routerID slightly more predictable. Removing the duplicates protects against bad user input or the operator getting into a bad state.